src/continuous-calculator/continuousPreTest.js

```diff
@@ -0,0 +1,331 @@
+import {TextField, InputAdornment, FormControl, Box, Tooltip, Button} from "@mui/material";
+import "./continuousPreTest.css"
+import jstat from 'jstat';
+import React, {useState} from "react";
+
+export default function ContinuousPreTestCalculator() {
+    const [isDetailed, setDetail] = useState(true);
+    const [formData, setData] = useState({
+        ctrlTrafficPercentInput: 0,
+        varTrafficPercentInput: 0,
+        conBaseRevInput: 0,
+        desiredLiftInput: 0,
+        pooledStandardDeviationInput: 0,
+        confidenceLvlInput: 0,
+        statisticalPowerInput: 0,
+        dailyVisitorsInput: 0
+        
+    })
+
+    const groupSizeRatio = (formData.ctrlTrafficPercentInput * 0.01) / (formData.varTrafficPercentInput * 0.01);
+    const varGroupRevenue = formData.conBaseRevInput * (1 + (formData.desiredLiftInput * .01));
+    const revAbsoluteDiff = Math.abs(formData.conBaseRevInput - varGroupRevenue);
+    const varSampleSize = (1 + (1 / groupSizeRatio)) * (Math.pow( formData.pooledStandardDeviationInput * ( (jstat.normal.inv(((1 - ((1 - (formData.confidenceLvlInput * 0.01))) / 2), 0, 1)) + 
+        jstat.normal.inv((formData.statisticalPowerInput * 0.01), 0, 1)) / (formData.conBaseRevInput - varGroupRevenue))),2);
+    const conSampleSize = (varSampleSize * groupSizeRatio);
```

Comment:
These need to be changed as they don't look like the correct calculations, just making a note for future ref